File: stress/t/ctl_classify.f
      PROGRAM CTLCLASS
      INTEGER*8 X,I,NEG,ZERO,POS,BIG
      NEG=0
      ZERO=0
      POS=0
      BIG=0
      I=0
   10 IF (I.GE.20) GOTO 20
      X=MOD(I*7,11_8)-5
      IF (X.LT.0) THEN
        NEG=NEG+1
      ELSE IF (X.EQ.0) THEN
        ZERO=ZERO+1
      ELSE IF (X.GT.3) THEN
        BIG=BIG+1
      ELSE
        POS=POS+1
      END IF
      I=I+1
      GOTO 10
   20 CONTINUE
      WRITE(*,'(A,I0)') 'negc=',NEG
      WRITE(*,'(A,I0)') 'zero=',ZERO
      WRITE(*,'(A,I0)') 'pos=',POS
      WRITE(*,'(A,I0)') 'big=',BIG
      END

File: stress/t/ctl_classify.c
#include <stdio.h>
#include <stdint.h>
int main(void){
  int64_t x=0,i=0,negc=0,zero=0,pos=0,big=0;
  for(i=0;i<20;i=i+1){
    x=(i*7)%11-5;
    if(x<0){ negc=negc+1; }
    else { if(x==0){ zero=zero+1; } else { if(x>3){ big=big+1; } else { pos=pos+1; } } }
  }
  printf("negc="); printf("%ld",(long)negc); printf("\n");
  printf("zero="); printf("%ld",(long)zero); printf("\n");
  printf("pos="); printf("%ld",(long)pos); printf("\n");
  printf("big="); printf("%ld",(long)big); printf("\n");
  return 0;
}
